renderer/main.js

```diff
@@ -1213,11 +1213,7 @@ function openTorrentContextMenu (infoHash) {
 }
 
 function getTorrentPath (torrentSummary) {
-  var itemPath = path.join(torrentSummary.path, torrentSummary.files[0].path)
-  if (torrentSummary.files.length > 1) {
-    itemPath = path.dirname(itemPath)
-  }
-  return itemPath
+  return path.join(torrentSummary.path, torrentSummary.name)
```

Comment:
Just realized this probably is because multi file torrents must be within a folder path?
I think deleting each file individually sound like the best solution, but what about the empty folders remaining? If we just remove them when they are empty the Downloads folder might still be removed?